pkg/utils/names/names.go

```diff
@@ -18,8 +18,21 @@ package names
 
 import (
 	"fmt"
+	"strings"
+)
+
+const (
+	FloatingAddressIPClaimNameSuffix = "floating-address"
```

Comment:
openstack doesn't have this term, will 
floating-ip-address or something simliar be better?

Reply:
Reasonable, that should probably be done everywhere e.g `FloatingAddressFromPoolReady` -> `FloatingIPAddressFromPoolReady` right?

I think `FloatingIPAddress..` is preferable over `FloatingIP..` or `FloatingAddress..` here imo, since it's an openstack floating ip added as machine addresses